### PR TITLE
Correction chocolatey install Script

### LINKED ISSUE #3651: Update Chocolatey install script to work for `v0.11.0`
As of now it doesn't work for `v0.11.0-rc.2`. See the logs below. I made some minimal changes to try installing `v0.11.0-rc.2` - I just changed the version and SHA sum in these places

https://github.com/vmware-tanzu/community-edition/blob/b7211c0189a081653b60551f56784ad71fb4677b/hack/choco/tools/chocolateyinstall.ps1#L5

https://github.com/vmware-tanzu/community-edition/blob/b7211c0189a081653b60551f56784ad71fb4677b/hack/choco/tools/chocolateyinstall.ps1#L12

I did NOT change any other file / content like nuspec etc https://github.com/vmware-tanzu/community-edition/blob/b7211c0189a081653b60551f56784ad71fb4677b/hack/choco/tanzu-community-edition.nuspec#L6 where changes are required for `v0.11.0` but since I was just testing, I didn't change those

```
PS C:\Users\Administrator> choco
Chocolatey v1.0.0
Please run 'choco -?' or 'choco <command> -?' for help menu.
PS C:\Users\Administrator>
PS C:\Users\Administrator> mkdir $HOME\tce-pkg


    Directory: C:\Users\Administrator


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         3/22/2022   9:39 AM                tce-pkg


PS C:\Users\Administrator> ls


    Directory: C:\Users\Administrator


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-r---         3/22/2022   9:03 AM                3D Objects
d-r---         3/22/2022   9:03 AM                Contacts
d-r---         3/22/2022   9:03 AM                Desktop
d-r---         3/22/2022   9:18 AM                Documents
d-r---         3/22/2022   9:33 AM                Downloads
d-r---         3/22/2022   9:03 AM                Favorites
d-r---         3/22/2022   9:03 AM                Links
d-r---         3/22/2022   9:03 AM                Music
d-r---         3/22/2022   9:03 AM                Pictures
d-r---         3/22/2022   9:03 AM                Saved Games
d-r---         3/22/2022   9:03 AM                Searches
d-----         3/22/2022   9:39 AM                tce-pkg
d-r---         3/22/2022   9:03 AM                Videos


PS C:\Users\Administrator> cd .\Downloads\
PS C:\Users\Administrator\Downloads> ls


    Directory: C:\Users\Administrator\Downloads


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         3/22/2022   9:33 AM       40647186 community-edition-main.zip


PS C:\Users\Administrator\Downloads> ls


    Directory: C:\Users\Administrator\Downloads


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         3/22/2022   9:40 AM                community-edition-main
-a----         3/22/2022   9:33 AM       40647186 community-edition-main.zip


PS C:\Users\Administrator\Downloads> cd .\community-edition-main\community-edition-main\
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main> ls


    Directory: C:\Users\Administrator\Downloads\community-edition-main\community-edition-main


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         3/22/2022   9:40 AM                .github
d-----         3/22/2022   9:40 AM                addons
d-----         3/22/2022   9:40 AM                cli
d-----         3/22/2022   9:40 AM                docs
d-----         3/22/2022   9:41 AM                hack
d-----         3/22/2022   9:41 AM                logos
d-----         3/22/2022   9:41 AM                test
-a----         3/22/2022   9:40 AM            124 .gitattributes
-a----         3/22/2022   9:40 AM            838 .gitignore
-a----         3/22/2022   9:40 AM           3098 .golangci.yaml
-a----         3/22/2022   9:40 AM            167 .markdownlintrc
-a----         3/22/2022   9:40 AM           1476 ADOPTERS.md
-a----         3/22/2022   9:40 AM           3466 CODEOWNERS
-a----         3/22/2022   9:40 AM           5470 CODE_OF_CONDUCT.md
-a----         3/22/2022   9:40 AM            203 CONTRIBUTING.md
-a----         3/22/2022   9:40 AM            282 go.mod
-a----         3/22/2022   9:40 AM          57082 go.sum
-a----         3/22/2022   9:40 AM           3837 GOVERNANCE.md
-a----         3/22/2022   9:40 AM          10143 LICENSE
-a----         3/22/2022   9:40 AM           2252 MAINTAINERS.md
-a----         3/22/2022   9:40 AM          19512 Makefile
-a----         3/22/2022   9:40 AM           1113 METRICS.md
-a----         3/22/2022   9:40 AM           9535 NOTICE
-a----         3/22/2022   9:40 AM           8675 README.md
-a----         3/22/2022   9:40 AM           3404 release.yml
-a----         3/22/2022   9:40 AM           6922 ROADMAP.md
-a----         3/22/2022   9:40 AM          10703 SECURITY.md


PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main> cd .\hack\check\
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\check> cd ..
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack> cd .\choco\
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco> ls


    Directory: C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         3/22/2022   9:41 AM                test
d-----         3/22/2022   9:41 AM                tools
-a----         3/22/2022   9:41 AM            914 packpush.ps1
-a----         3/22/2022   9:41 AM           5572 README.md
-a----         3/22/2022   9:41 AM           1712 tanzu-community-edition.nuspec
-a----         3/22/2022   9:41 AM           2553 update-choco-package.sh

PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco> cat .\tools\chocolateyinstall.ps1
# Copyright 2020-2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

$ErrorActionPreference = 'Stop';
$releaseVersion = 'v0.11.0-rc.2'
$packageName = 'tanzu-community-edition'
$packageFullName = "tce-windows-amd64-$releaseVersion"
$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
# This line is for local testing
#$url64 = "C:\Users\...\tce-windows-amd64-${releaseVersion}.zip"
$url64 = "https://github.com/vmware-tanzu/community-edition/releases/download/${releaseVersion}/tce-windows-amd64-${releaseVersion}.zip"
$checksum64 = 'df40dcc40539a1ac2d83e054da1b15edb4b8ba8aa3ce9457e2c3e5bb9dafd3bf'
$checksumType64 = 'sha256'

$packageArgs = @{
    packageName    = $packageName
    unzipLocation  = $toolsDir
    url64bit       = $url64

    softwareName   = 'tanzu-community-edition'

    checksum64     = $checksum64
    checksumType64 = $checksumType64
}

function Test-Prereqs {
    # Since Windows users can install docker and kubectl a lot of different ways,
    # just see if the executables are there.
    Write-Host "`nChecking Prerequisites" -ForegroundColor Green

    if ( -not (Get-Command docker -ErrorAction SilentlyContinue)) {
        Write-Host -ForegroundColor Red "  - Docker CLI not present! This is required to create bootstrap clusters`n"
    }
    else {
        Write-Host "  - Docker CLI found, proceeding" -ForegroundColor Cyan
    }

    if ( -not (Get-Command kubectl -ErrorAction SilentlyContinue)) {
        Write-Host -ForegroundColor Red "  - kubectl not present! This is required to create bootstrap clusters`n"
    }
    else {
        Write-Host "  - kubectl found, proceeding`n" -ForegroundColor Cyan
    }
}

function Install-TanzuEnvironment {
    # important locations
    # XDG_DATA_HOME -> LOCALAPPDATA on Windows
    $PluginDir = "${env:LOCALAPPDATA}\tanzu-cli"
    $CacheLocation = "${HOME}\.cache\tanzu\catalog.yaml"
    $CLIConfigLocation = "${HOME}\.config\tanzu\config.yaml"
    $CompatabilityLocation = "${HOME}\.config\tanzu\tkg\compatibility\tkg-compatibility.yaml"

    Write-Host "`nStarted tanzu CLI environment setup" -ForegroundColor Green

    ## begin env clean up ##

    # if an existing compatibility file exists, remove it; the cli will redownload it
    if (Test-Path -Path $CompatabilityLocation -PathType Leaf) {
        Remove-Item -Path ${CompatabilityLocation} -Force
        Write-Host "  - Removed stale compatibility file at ${CompatabilityLocation}" -ForegroundColor Cyan
    }

    # if an existing config file exists, remove it in favor of a newly initialized one
    if (Test-Path -Path $CLIConfigLocation -PathType Leaf) {
        Remove-Item -Path ${CLIConfigLocation} -Force
        Write-Host "  - Removed existing CLI config at ${CLIConfigLocation}" -ForegroundColor Cyan
    }

    # if plugin cache exists, remove it; this ensures stale commands don't show up when running tanzu
    if (Test-Path -Path $CacheLocation -PathType Leaf) {
        Remove-Item -Path ${CacheLocation} -Force
        Write-Host "  - Removed existing tanzu plugin cache file at ${CacheLocation}" -ForegroundColor Cyan
    }

    ## end env clean up ##

    ## begin env setup ##

    # create the plugin directory for tanzu CLI
    New-Item -Path ${PluginDir} -ItemType directory -Force | Out-Null
    Write-Host "  - Created CLI plugin directory at ${pluginDir}" -ForegroundColor Cyan

    # for every plugin (syntax == "tanzu-*"), move it to ${XDG_DATA_HOME}/tanzu-cli
    # this is where tanzu CLI will lookup the plugin to wire into its command
    Get-ChildItem -Path "${toolsDir}\${packageFullName}\bin\tanzu-*" -Recurse | Move-Item -Destination ${PluginDir} -Force
    Write-Host "  - Moved CLI plugins to ${pluginDir}" -ForegroundColor Cyan

    # initialize CLI and add TCE plugin repo (bucket)
    # Note that we use the toolsDir path because chocolatey doesn't put
    # binaries on the $PATH until _after_ the install script runs.
    $tanzuExe = "${toolsDir}\${packageFullName}\bin\tanzu.exe"

    # The & allows execution of a binary stored in a variable.
    Write-Host "  - Initializing Tanzu configuration" -ForegroundColor Cyan
    & $tanzuExe init | Out-Null
    & $tanzuExe plugin repo add --name tce --gcp-bucket-name tce-tanzu-cli-plugins --gcp-root-path artifacts
    & $tanzuExe plugin repo add --name core-admin --gcp-bucket-name tce-tanzu-cli-framework-admin --gcp-root-path artifacts-admin

}
Test-Prereqs

# this is a built-in function, read https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage
Install-ChocolateyZipPackage @packageArgs

Install-TanzuEnvironment
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco>

PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco> .\packpush.ps1
Remove-Item : Cannot find path 'C:\Users\Administrator\AppData\Local\Temp\chocolatey\tanzu-community-edition\' because
it does not exist.
At C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco\packpush.ps1:13 char:1
+ Remove-Item ${env:LOCALAPPDATA}\Temp\chocolatey\tanzu-community-editi ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Admini...munity-edition\:String) [Remove-Item], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand

Chocolatey v1.0.0
Attempting to build package from 'tanzu-community-edition.nuspec'.
Successfully created package 'C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco\tanzu-community-edition.0.10.0.nupkg'
Chocolatey v1.0.0
Attempting to push tanzu-community-edition.0.10.0.nupkg to C:\Users\Administrator\tce-pkg
tanzu-community-edition 0.10.0 was pushed successfully to C:\Users\Administrator\tce-pkg

PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco> choco install tanzu-community-edition --source $HOME\tce-pkg
Chocolatey v1.0.0
Installing the following packages:
tanzu-community-edition
By installing, you accept licenses for the packages.

tanzu-community-edition v0.10.0
tanzu-community-edition package files install completed. Performing other installation steps.
The package tanzu-community-edition wants to run 'chocolateyinstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): A


Checking Prerequisites
  - Docker CLI found, proceeding
  - kubectl not present! This is required to create bootstrap clusters

Downloading tanzu-community-edition 64 bit
  from 'https://github.com/vmware-tanzu/community-edition/releases/download/v0.11.0-rc.2/tce-windows-amd64-v0.11.0-rc.2.zip'
Progress: 100% - Completed download of C:\Users\Administrator\AppData\Local\Temp\chocolatey\tanzu-community-edition\0.10.0\tce-windows-amd64-v0.11.0-rc.2.zip (246.22 MB).
Download of tce-windows-amd64-v0.11.0-rc.2.zip (246.22 MB) completed.
Hashes match.
Extracting C:\Users\Administrator\AppData\Local\Temp\chocolatey\tanzu-community-edition\0.10.0\tce-windows-amd64-v0.11.0-rc.2.zip to C:\ProgramData\chocolatey\lib\tanzu-community-edition\tools...
C:\ProgramData\chocolatey\lib\tanzu-community-edition\tools

Started tanzu CLI environment setup
  - Created CLI plugin directory at C:\Users\Administrator\AppData\Local\tanzu-cli
ERROR: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Cannot find path 'C:\ProgramData\chocolatey\lib\tanzu-community-edition\tools\tce-windows-amd64-v0.11.0-rc.2\bin\' because it does not exist.
The install of tanzu-community-edition was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\tanzu-community-edition\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - tanzu-community-edition (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\tanzu-community-edition\tools\chocolateyinstall.ps1'.
 See log for details.
PS C:\Users\Administrator\Downloads\community-edition-main\community-edition-main\hack\choco>
```

### PR DESCRIPTION
**Pull request by aman556**
Signed-off-by: Aman Sharma <amansh@vmware.com>

## What this PR does / why we need it

This PR is doing the correction according to the new installation in `v0.11.0`. 
## Which issue(s) this PR fixes

Fixes: [#3651](https://github.com/vmware-tanzu/community-edition/issues/3651)

### CODE REVIEW

**Comment by karuppiah7890** [comment]
@joshrosso We are noticing an error while installing using choco with updated package code that will work for `v0.11.0`

Error - #3735

**Comment by joshrosso** [comment]
> @joshrosso We are noticing an error while installing using choco with updated package code that will work for `v0.11.0`
> 
> Error - #3735
> 
> How do we wanna go about resolving this issue? As in, this happens in the choco script when it runs `tanzu init` over here -
> 
> https://github.com/vmware-tanzu/community-edition/blob/e8b1bf17ab58822ed27594553d44b946fffccad6/hack/choco/tools/chocolateyinstall.ps1#L96
> 
> And the TCE chocolatey installation fails

I'll take a look at this today. Thanks.

**Comment by dvonthenen** [comment]
this works for me....
```
C:\Users\vonthd\Documents>cd tce-windows-amd64-v0.11.0

C:\Users\vonthd\Documents\tce-windows-amd64-v0.11.0>dir
 Volume in drive C has no label.
 Volume Serial Number is 3C19-A077

 Directory of C:\Users\vonthd\Documents\tce-windows-amd64-v0.11.0

03/28/2022  11:46 AM    <DIR>          .
03/28/2022  11:46 AM    <DIR>          ..
03/28/2022  11:47 AM    <DIR>          default-local
03/24/2022  12:00 PM             1,803 install.bat
03/24/2022  12:10 PM        59,755,240 tanzu.exe
03/24/2022  12:00 PM             2,604 uninstall.bat
               3 File(s)     59,759,647 bytes
               3 Dir(s)  26,483,478,528 bytes free

C:\Users\vonthd\Documents\tce-windows-amd64-v0.11.0>uninstall.bat
Start    : 11:47:37.05
Finish   : 11:47:37.06
         ---------------
Duration : 00:00:00,01
"Uninstall complete!"
"Remove C:\Program Files\tanzu from your system's PATH."

C:\Users\vonthd\Documents\tce-windows-amd64-v0.11.0>install.bat
        1 file(s) copied.
default-local\discovery\standalone\builder.yaml
default-local\discovery\standalone\cluster.yaml
default-local\discovery\standalone\codegen.yaml
default-local\discovery\standalone\conformance.yaml
default-local\discovery\standalone\diagnostics.yaml
default-local\discovery\standalone\kubernetes-release.yaml
default-local\discovery\standalone\login.yaml
default-local\discovery\standalone\management-cluster.yaml
default-local\discovery\standalone\package.yaml
default-local\discovery\standalone\pinniped-auth.yaml
default-local\discovery\standalone\secret.yaml
default-local\discovery\standalone\unmanaged-cluster.yaml
default-local\distribution\windows\amd64\cli\builder\v0.11.2\tanzu-builder-windows_amd64.exe
default-local\distribution\windows\amd64\cli\cluster\v0.11.2\tanzu-cluster-windows_amd64.exe
default-local\distribution\windows\amd64\cli\codegen\v0.11.2\tanzu-codegen-windows_amd64.exe
default-local\distribution\windows\amd64\cli\conformance\v0.11.0\tanzu-conformance-windows_amd64.exe
default-local\distribution\windows\amd64\cli\diagnostics\v0.11.0\tanzu-diagnostics-windows_amd64.exe
default-local\distribution\windows\amd64\cli\kubernetes-release\v0.11.2\tanzu-kubernetes-release-windows_amd64.exe
default-local\distribution\windows\amd64\cli\login\v0.11.2\tanzu-login-windows_amd64.exe
default-local\distribution\windows\amd64\cli\management-cluster\v0.11.2\tanzu-management-cluster-windows_amd64.exe
default-local\distribution\windows\amd64\cli\package\v0.11.2\tanzu-package-windows_amd64.exe
default-local\distribution\windows\amd64\cli\pinniped-auth\v0.11.2\tanzu-pinniped-auth-windows_amd64.exe
default-local\distribution\windows\amd64\cli\secret\v0.11.2\tanzu-secret-windows_amd64.exe
default-local\distribution\windows\amd64\cli\unmanaged-cluster\v0.11.0\tanzu-unmanaged-cluster-windows_amd64.exe
24 File(s) copied
Installing plugin 'builder:v0.11.2'
✔  successfully installed 'builder' plugin
Installing plugin 'codegen:v0.11.2'
✔  successfully installed 'codegen' plugin
Installing plugin 'cluster:v0.11.2'
✔  successfully installed 'cluster' plugin
Installing plugin 'kubernetes-release:v0.11.2'
✔  successfully installed 'kubernetes-release' plugin
Installing plugin 'login:v0.11.2'
✔  successfully installed 'login' plugin
Installing plugin 'management-cluster:v0.11.2'
✔  successfully installed 'management-cluster' plugin
Installing plugin 'package:v0.11.2'
✔  successfully installed 'package' plugin
Installing plugin 'pinniped-auth:v0.11.2'
✔  successfully installed 'pinniped-auth' plugin
Installing plugin 'secret:v0.11.2'
✔  successfully installed 'secret' plugin
Installing plugin 'conformance:v0.11.0'
✔  successfully installed 'conformance' plugin
Installing plugin 'diagnostics:v0.11.0'
✔  successfully installed 'diagnostics' plugin
Installing plugin 'unmanaged-cluster:v0.11.0'
✔  successfully installed 'unmanaged-cluster' plugin
        1 file(s) copied.
"Installation complete!"
"Please add C:\Program Files\tanzu permanently into your system's PATH."

C:\Users\vonthd\Documents\tce-windows-amd64-v0.11.0>
```

**Comment by joshrosso** [comment]
Verified `choco` failure is unrelated to https://github.com/vmware-tanzu/community-edition/issues/3735

Will do a better write-up in a sec, but for now, here is what is going on:

https://user-images.githubusercontent.com/6200057/160484271-4b5cbdaa-a96c-43b5-aaa0-ea6a0c43c146.mp4

**Comment by joshrosso** [comment]
Added https://github.com/vmware-tanzu/community-edition/pull/3736/commits/65d4e5effe2dc5c5a1d20b2967cdc245bff70e47 to correct the install.

**Review by davidvonthenen — APPROVED** [review]
Will merge for release tomorrow

**Review by joshrosso — APPROVED** [review]

**Comment by karuppiah7890** [comment]
@dvonthenen We can already merge and do the `choco push` manually from our local to push the choco package and start the review process. What do you think?

**Comment by karuppiah7890** [comment]
@joshrosso Thanks 🙏 We did notice the issue around plugin directory already existing. It was weird to notice `-Force` being used but it still not working out. Good idea to just remove the directory entirely if it already exists

File: hack/choco/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -93,9 +99,13 @@ function Install-TanzuEnvironment {
 
     # The & allows execution of a binary stored in a variable.
     Write-Host "  - Initializing Tanzu configuration" -ForegroundColor Cyan
-    & $tanzuExe init | Out-Null
-    & $tanzuExe plugin repo add --name tce --gcp-bucket-name tce-tanzu-cli-plugins --gcp-root-path artifacts
-    & $tanzuExe plugin repo add --name core-admin --gcp-bucket-name tce-tanzu-cli-framework-admin --gcp-root-path artifacts-admin
```
Comment (karuppiah7890) [inline hack/choco/tools/chocolateyinstall.ps1:98]: We don't wanna add the TCE and core-admin plugin repositories after `tanzu init`?

File: hack/choco/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -2,14 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 $ErrorActionPreference = 'Stop';
-$releaseVersion = 'v0.10.0'
+$releaseVersion = 'v0.11.0-rc.2'
```
Comment (karuppiah7890) [inline hack/choco/tools/chocolateyinstall.ps1:5]: We gotta change this to `v0.11.0` 😅

File: hack/choco/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
 $packageName = 'tanzu-community-edition'
 $packageFullName = "tce-windows-amd64-$releaseVersion"
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 # This line is for local testing
 #$url64 = "C:\Users\...\tce-windows-amd64-${releaseVersion}.zip"
 $url64 = "https://github.com/vmware-tanzu/community-edition/releases/download/${releaseVersion}/tce-windows-amd64-${releaseVersion}.zip"
-$checksum64 = 'b4f868c9bef84c8824287c54424a603ea05f84b7894a45ba0a44d0ee7d7f1ed9'
+$checksum64 = 'df40dcc40539a1ac2d83e054da1b15edb4b8ba8aa3ce9457e2c3e5bb9dafd3bf'
```
Comment (karuppiah7890) [inline hack/choco/tools/chocolateyinstall.ps1:12]: We need to update the checksum to `2a859d12f65fee5d1f6cced16484a6398e3e3190064185608eb59b6529ce054b` which is the checksum for `v0.11.0`. The current one is for RC.2

File: hack/choco/tools/chocolateyinstall.ps1 (hunk 2 above)
Comment (dvonthenen) [reply hack/choco/tools/chocolateyinstall.ps1:5]: There is another PR doing this:
https://github.com/vmware-tanzu/community-edition/pull/3727
https://github.com/vmware-tanzu/community-edition/pull/3728

**Pull request merged**